partner-onboarder/install.sh

```diff
@@ -54,6 +54,23 @@ function installing_onboarder() {
     --version $CHART_VERSION
 
     echo Reports are moved to S3 under onboarder bucket
+
+    read -p "The below script copies esignet misp license key to secrets, Please reply yes only if esignet partner is onboarded (Y/n) " yn;
```

Comment:
what is the use of this check?

Reply:
yes sir the check is not needed.. gave it initially since i added this in mosip-infra's partner onboarder
updated the script